shared/chat/inbox/index.desktop.js

```diff
@@ -175,6 +129,7 @@ class Inbox extends PureComponent<Props, State> {
             onSelectUp={this.props.onSelectUp}
             onSelectDown={this.props.onSelectDown}
           />
+          // TODO: there is no longer showNewConversation
```

Comment:
This shows up! 
<img width="316" alt="screen shot 2017-11-01 at 10 27 15 am" src="https://user-images.githubusercontent.com/594035/32287823-40c072a4-beef-11e7-9a67-8190e588f302.png">

Reply:
HA! whoops!